### PR TITLE
feat: add libselinux, libsepol, pcre2 and libcap

### PR DESCRIPTION
**Pull request by dsseng**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -60,17 +60,19 @@ TARGETS += iptables
 TARGETS += ipxe
 TARGETS += kmod
 TARGETS += libaio
-TARGETS += libattr
```
Comment (smira) [inline Makefile:63]: this looks like a bug in previous commits, but this should stay

**Review by smira — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (dsseng) [reply Makefile:63]: I did a make rekres. Should I just add `TARGETS += libattr` back?

**Comment by dsseng** [comment]
@smira please also check libcap, decided to add it here as well

**Pull request merged**